pom.xml

```diff
@@ -101,6 +107,27 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<dependency>
+				<groupId>org.apache.kafka</groupId>
+				<artifactId>kafka_2.11</artifactId>
+				<classifier>test</classifier>
+				<scope>test</scope>
+				<version>${kafka.version}</version>
+				<exclusions>
+					<exclusion>
+						<groupId>jline</groupId>
+						<artifactId>jline</artifactId>
+					</exclusion>
+					<exclusion>
+						<groupId>org.slf4j</groupId>
+						<artifactId>slf4j-log4j12</artifactId>
+					</exclusion>
+					<exclusion>
+						<groupId>log4j</groupId>
+						<artifactId>log4j</artifactId>
+					</exclusion>
+				</exclusions>
+			</dependency>
```

Comment:
Why is this one in the parent but kafka itself is in the child poms?

Reply:
The ones in the parent pom are under the `dependencyManagment` section (Both kafka and the one with the `test` classifier). They both are in the relevant `dependencies` configuration in the child poms. Is that what you meant?

Reply:
Yeah - I don't really know maven well enough to comment; it just looks a bit odd. I'll take your word if it's correct.